resources/style.css

```diff
@@ -7,28 +7,59 @@ body {
   height: 100vh;
 }
 
+nav.navbar {
+  background-color: rgba(52, 51, 50, 0.8);
+  position: fixed;
```

Comment:
Unnecessary since there is the `fixed-top` class
```suggestion
```

Reply:
Thanks, forgot to remove after I found out that was a thing